### ISSUE TITLE
eea.jquery can be installed in the Plone 5.1 control panel

### ISSUE THREAD

**Issue by fredvd** [eea/eea.jquery#13]
I'm testing eea.facetednavigation in Plone 5.1 and I found several extra packages in the Control Panel Add'ons form.

From what I see in the package eea.jquery is primarily meant to be installed by developers that depend on a number of eea.jquery profiles in their own addon's metadata.xml . 

Shouldn't eea.jquery be hidden completely in the Control Panel from being installable?  https://docs.plone.org/develop/addons/upgrade_to_51.html#inoninstallable

![installable](https://user-images.githubusercontent.com/394784/40549706-7c5e1358-6038-11e8-918d-626cb91bcc19.png)

**Comment by avoinea** [comment]
@fredvd indeed, it shouldn't bee visible, but I think nobody took care of it when migrated from Plone 4. 

A PR would be more than welcomed. Thank you!
